server/php-engine.cpp

```diff
@@ -1508,7 +1509,8 @@ static void generic_event_loop(WorkerType worker_type, bool init_and_listen_rpc_
   if (no_sql) {
     sql_target_id = -1;
   } else {
-    sql_target_id = get_target("localhost", db_port, &db_ct);
+    fprintf(stdout, "mysql host: %s; port: %d\n", db_host, db_port);
```

Comment:
If you want to log something here, use `kprintf` for this. It adds some useful output like pid, time and so on

Reply:
Ok, fixed